helix-view/src/editor.rs

```diff
@@ -232,6 +232,8 @@ pub struct LspConfig {
     pub auto_signature_help: bool,
     /// Display docs under signature help popup
     pub display_signature_help_docs: bool,
+    /// Display context of current cursor line if it is outside the view.
+    pub context: bool,
```

Comment:
Maybe move to the `editor::Config` struct and go with a new category: `sticky_context: StickyContext`.

- enable: to enable sticky context feature (default: false)
- min_window_height: disable sticky context when the window is too small (default: ?)

Reply:
Moved to `editor::Config` but left it as boolean config option for now. I am not sure if `min_window_height` is necessary as the context usually spans only a few lines. What do you thing?

Reply:
I’m just thinking this can be useful when the editing some file in a small window. What do you think?

Reply:
Maybe a keybind to toggle it? That way it can be decided per case.